LEGO1/lego/legoomni/include/legocarbuildpresenter.h

```diff
@@ -116,29 +123,29 @@ class LegoCarBuildAnimPresenter : public LegoAnimPresenter {
 private:
 	void Beta10Inline0x100733d0();
 
-	MxU16 m_unk0xbc; // 0xbc
+	ShelfState m_shelfState; // 0xbc
```

Comment:
The type of this needs to remain `MxU16`. Using `ShelfState` will cause it to be an `int`